### DISCUSSION TITLE
Formatting with saving excel file

### DISCUSSION THREAD

**Discussion by HAK-AL** [aws/aws-sdk-pandas#1336] [Q&A]
Hi,
Question is can we do formatting with excel before saving to S3 via openpyxl or xlsxwriter if yes, can share code snippet?

**Comment by kukushking** [comment]
Wrangler is basically passing the call to [pandas.DataFrame.to_excel](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_excel.html) so you can pass the same args to wrangler call and control the behavior of the excel writer.
